model/Core/Classes/X-Collection.md

```diff
@@ -0,0 +1,30 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# X-Collection
+
+## Summary
+
+A collection of SPDX Elements that could potentially be serialized.
+
+## Description
+
+The X-Collection provides a convenient way to express information about collections of SPDX Elements that could potentially be serialized as complete
+units (e.g., all in-scope SPDX data within a single JSON-LD file). X-Collection is indpendent of any particular serialization format or instance.
```

Comment:
```suggestion
units (e.g., all in-scope SPDX data within a single JSON-LD file). X-Collection is independent of any particular serialization format or instance.
```

Reply:
@goneall LOL. It took me three reads through to notice my typo you corrected. I will fix it.